test/powershell/Language/Scripting/I18n.Tests.ps1

```diff
@@ -52,6 +52,7 @@ Describe 'Testing of script internationalization' -Tags "CI" {
 
         import-localizedData mydata -uiculture nl-NL -ea SilentlyContinue -ev ev
 
+        $ev | Should Not BeNullOrEmpty
         $ev[0].Exception.GetType() | Should Be System.Management.Automation.PSInvalidOperationException
```

Comment:
Sometimes it is clear from above code that '$result' is not empty. Ex.  `$result.Count | Should Be 2`